roles/rsyslog/templates/output_files.j2

```diff
@@ -42,12 +42,19 @@ ruleset(name="{{ __rsyslog_output.name }}"
 ruleset(name="{{ __rsyslog_output.name }}") {
 {%   endif %}
 {{ print_file_attrs(__rsyslog_output) -}}
+{%   set template = " ;RSYSLOG_TraditionalFileFormat"
```

Comment:
Is it possible to write it to make it more readable? Something like this but with all logic?
```
{% if __rsyslog_output.template | d("") == "traditional" %}
{%   set template  =   " ;RSYSLOG_TraditionalFileFormat" %}
{% endif %}
```
Same for roles/rsyslog/templates/output_remote_files.j2.

Reply:
> Is it possible to write it to make it more readable?

Unfortunately no, due to the way jinja does scoping.  https://jinja.palletsprojects.com/en/stable/templates/#list-of-builtin-filters
"Scoping Behavior

Please keep in mind that it is not possible to set variables inside a block and have them show up outside of it."

Reply:
Okay, I was looking at what I did in mssq, but indeed there I am doing it within a single block https://github.com/linux-system-roles/mssql/blob/main/templates/configure_ag.j2